app/src/hog.c

```diff
@@ -156,28 +157,120 @@ struct bt_conn *destination_connection() {
     return conn;
 }
 
+#define HOG_STACK_SIZE 512
+#define HOG_PRIORITY 5
+
+K_THREAD_STACK_DEFINE(hog_q_stack, HOG_STACK_SIZE);
+
+struct k_work_q hog_work_q;
+
+#define HOG_REPORT_QUEUE_SIZE 5
```

Comment:
As [discussed](https://discord.com/channels/719497620560543766/784523018050469939/799337259840045167), we should probably increase the queue size.

Reply:
Bumped the default in the Kconfig. Made two values, one for keyboard, one for consumer, and left the consumer one smaller. That should have less churn, but can be easily bumped later.